test/command/suite/select/function/snippet/many_keywords.test

```diff
@@ -0,0 +1,42 @@
+table_create Entries TABLE_NO_KEY
+column_create Entries content COLUMN_SCALAR ShortText
+
+load --table Entries
+[
+{"content": "groonga and MySQL and PosrgreSQL"}
+]
+
+select Entries \
+  --output_columns ' \
+  snippet(content, \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
+  "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", "SQL", \
```

Comment:
Could you use different keywords to confirm all specified keywords are used?

Reply:
Fixed.